src/lib/CoinbasePro/Authenticated/Withdrawal.hs

```diff
@@ -22,6 +22,8 @@ import           Data.UUID                          (UUID)
 
 import           CoinbasePro.Authenticated.Accounts (AccountId)
 import           CoinbasePro.Authenticated.Payment  (PaymentMethodId)
+import           Control.Applicative
+import Text.Read (readMaybe)
```

Comment:
Minor nitpick: do you mind running stylish Haskell so it lines up with the other imports? Thanks!